composer.json

```diff
@@ -15,7 +15,8 @@
         "source": "https://github.com/jasny/Config"
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "aws/aws-sdk-php": "2.*"
```

Comment:
Let's [suggest](https://getcomposer.org/doc/04-schema.md#suggest) aws/aws-sdk-php and set it as `require-dev`.